fractal_server/app/routes/api/v2/_aux_functions_tasks.py

```diff
@@ -211,6 +220,26 @@ async def _get_valid_user_group_id(
     return user_group_id
 
 
+async def _get_collection_status_message(
+    task_group: TaskGroupV2, db: AsyncSession
+) -> str:
+    res = await db.execute(
+        select(CollectionStateV2).where(
+            CollectionStateV2.taskgroupv2_id == task_group.id
+        )
+    )
+    state = res.scalars().one_or_none()
```

Comment:
Let's protect this with an `try/except` check.
If there are more than one, `msg` can say that (`Invalid state: found more than one CollectionStateV2 associated to taskgroup 43`).

Reply:
Even simpler: use an `if`